### ISSUE TITLE
Clique Community Persistence: A Topological Visual Analysis Approach for Complex Networks

### ISSUE THREAD

**Issue by Submanifold** [steveharoz/open-access-vis#38]
Dear Steve,

thank you so much for this service! I have finally added *all* information about this project on OSF.io. Would you kindly include the relevant URIs?

- All data: https://osf.io/rdktg/
- Paper (PDF): https://mfr.osf.io/render?url=https://osf.io/td973/?action=download%26mode=render
- Supplementary materials (PDF): https://mfr.osf.io/render?url=https://osf.io/e8dmp/?action=download%26mode=render
- Additional explanations: https://mfr.osf.io/render?url=https://osf.io/ygd6m/?action=download%26mode=render

Hope that I am using this service correcetly---this is my first project on OSF, inspired by your initiative. Thanks for your efforts!

Best,
  Bastian

**Comment by steveharoz** [comment]
Thanks Bastian. The paper is updated. And I've put the OSF repository as the materials. The data category is for experiment results such as experiment subject responses or algorithmic runtime results. I'm not sure the data folder here fits into that. Feel free to comment in this issue if you think anything should change.

**Comment by Submanifold** [comment]
Hi Steve! I realize that I should have named the folder somewhat differently, but it actually contains the results of algorithmic runs as well as scripts to create them for yourself (in order to compare them with the ones we reported in the paper). I'll also add a link from the GitHub repository to the OSF one so that more people are capable of finding it.

**Comment by Submanifold** [comment]
Hi Steve, sorry to bother you again, but I wanted to ask whether you could add the OSF repository as a data repository of our paper, as well. I have uploaded the raw input files as well as the results of our analysis in order to make everything reproducible.

**Comment by steveharoz** [comment]
I took a look at the data folder, and I'm still not sure if it fits the criteria of experiment data rather than materials.

I don't see:
1. A data dictionary (how do I read the data?)
2. An explanation in the paper about how the data is used for an analysis (e.g., mean & SD of algorithm run times) or comparison (algorithm A vs algorithm B; simulation vs ground truth; algorithm vs human estimation; etc.). I see the conclusion mentions a contrast with existing methods, but I don't see a discussion of the comparison using the data.

I just want to make sure that I understand what's posted, as it seems to be quite outside the typical umbrella of experiment results.

**Comment by Submanifold** [comment]
OK, I understand that I was mistaken about the nature of these experimental data. I have added an updated README to the *Data* folder to explain what is in there. In short, I added:
- all networks that we analysed in our paper
- the results of this analysis in the form of so-called persistence diagrams
- further data calculated with our method (topological centrality) and a comparison between existing centrality measures
- comparisons between our method and existing methods in the form of distance matrices (one is generated with the old method, the other with our method)
- embeddings & glyphs to reproduce the figures in our paper

I also added detailed instructions and automated scripts for reproducing every figure and every table in the table. The results are added in order to make it possible to check the correctness of the calculations.

So, I was hoping that this (along with the supplied code) should make it possible to reproduce just about everything in the paper (and also try it out with new data, if possible).

Sorry for taking up so much time; I have to admit that *Open Science* is something new for me&mdash;but I'm very excited to try it out and see the benefits for other scientists.
